build

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
@gtaharaedmonds I don't know why this is modified, I think it might have to do with Docker issues from a while ago??